File: ruby/recursive_count.rb
def recursive_count(num = 0)
  # type your code in here
  counting(num)
end

# counting
def counting (count = 0)
  while count < 10
    puts count
    count += 1
  end
end


if __FILE__ == $PROGRAM_NAME
  recursive_count
end

# OPTIONAL
# Please add your pseudocode to this file
# And a written explanation of your solution

File: javascript/recursive_count.js
function recursiveCount(num = 0) {
  // type your code here
  counting(num)
}

const counting = (count = 0) => {
  while (count < 10) {
    console.log(count);
    ++count;
  }
}


if (require.main === module) {
  recursiveCount();
}

module.exports = recursiveCount;

// OPTIONAL
// Please add your pseudocode to this file
// And a written explanation of your solution
